docs/tutorials/cargo_tests/baked/build.rs

```diff
@@ -0,0 +1,27 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu::locid::langid;
+use icu_datagen::CldrLocaleSubset;
+use icu_datagen::SourceData;
+use icu_provider::marker::KeyedDataMarker;
```

Comment:
Maybe we should have a datagen prelude that reexports `langid` and `KeyedDataMarker` so that datagen can be called without two additional dependencies.

Reply:
https://github.com/unicode-org/icu4x/issues/2957

Reply:
You can now use it.

Reply:
Done